src/codeu/chat/client/core/View.java

```diff
@@ -150,11 +152,11 @@ public ServerInfo getInfo() {
       } else {
         // Communicate this error - the server did not respond with the type of
         // response we expected.
-        System.out.println("Error: The server did not respond with the type of response we expected. Line 146-149, View.java.");
+        new IOException().printStackTrace("ERROR: The server did not respond with the type of response we expected.");
       }
     } catch (Exception ex) {
       // Communicate this error - something went wrong with the connection.
-      System.out.println("Fatal error: Unexpected error occurs. Line 146-149, View.java" + ex.getMessage());
+      new UnexpectedException().printStackTrace("FATAL ERROR: Unexpected error occurs.");
```

Comment:
1. Why is this a fatal error?
2. An unexpected error could happen that would have nothing to do with the connection, but the one that got us here did. This line does not express that.
3. This line has the same error w/printStackTrace syntax as the previous line I corrected. It should be:
System.out.println("insert error message here");
ex.printStackTrace();